### PR TITLE
Unrestricts Underwear And Hair Based On Gender

### PR DESCRIPTION
**Pull request by Isy232**
## About The Pull Request

Changes the male and female hair styles to be able to be chosen by both male and female characters.

## Why It's Good For The Game

For if you want to have long hair as a male or short hair as a female.

~~Or you want to play as a degenerate fucking femboy~~

## Changelog
:cl:
add: Added femboys.
tweak: Underwear and hairstyle is no longer restricted by gender.
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
I feel like this is a personal attack for some odd reason and i dont like that

**Comment by PowerfulBacon** [comment]
It's already hard to distinguish between Male and female characters, hair and beard is one of the few ways to tell. Also males can take long hair anyway, I am pretty sure both hair lists are the same

**Comment by TeomanTheGreat** [comment]
Femboy hooters in space 😳

**Comment by eeSPee** [comment]
What? My sexy felinid officer can wear the based mankini? I love you, OP :3

**Comment by Moccha-Bee** [comment]
oh god pelase no, we already have ike playing as a trap we don't need any more traps!

**Comment by eeSPee** [comment]
This is why we can't let humans play god...

**Comment by Blueturbo47** [comment]
This gives new meaning to femboysimov

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -1336,23 +1336,23 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 				if("hair_style")
 					var/new_hair_style
 					if(gender == MALE)
-						new_hair_style = input(user, "Choose your character's hair style:", "Character Preference")  as null|anything in GLOB.hair_styles_male_list
+						new_hair_style = input(user, "Choose your character's hair style:", "Character Preference")  as null|anything in GLOB.hair_styles_list
 					else
-						new_hair_style = input(user, "Choose your character's hair style:", "Character Preference")  as null|anything in GLOB.hair_styles_female_list
+						new_hair_style = input(user, "Choose your character's hair style:", "Character Preference")  as null|anything in GLOB.hair_styles_list
 					if(new_hair_style)
```
Comment (francinum) [inline code/modules/client/preferences.dm:1342]: ...If you're going to make them query the same list, please take a second to dedupe your code.

Comment (Isy232) [reply code/modules/client/preferences.dm:1342]: Thanks for the review, if this PR does not get closed by a maintainer or get downvoted into the shadowrealm I will fix that.

**Comment by Isy232** [comment]
> It's already hard to distinguish between Male and female characters, hair and beard is one of the few ways to tell. Also males can take long hair anyway, I am pretty sure both hair lists are the same

Thats the point of this PR, to make it harder so you can play traps.

**Comment by ghost** [comment]
It eludes me as to why beestation hasn’t imploded yet

**Comment by PowerfulBacon** [comment]
This is a bad meme PR that fills the code with if elses that do the same thing either way. Doesn't even remove the other, now unused lists. Closing

**Pull request closed without merging**